simp_le.py

```diff
@@ -850,6 +850,10 @@ def create_parser():
         '--reuse_key', action='store_true', default=False,
         help='Reuse private key if it was previously persisted.',
     )
+    io_group.add_argument(
+        '--renew_different_san', action='store_true', default=False,
+        help='Don\'t abort if the SANs changed'
```

Comment:
What about something like:
"Allow replacement of a certificates with different SANs" or
"Force certificate renewal when domains are modified"

That should make it clearer what happens instead. Some suggested option names: `--allow_different_san` or `--allow_domain_changes`